movie-blog/src/components/components/contained/Filter.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from 'react';
+import Container from "../../elements/Container";
+import Label from "../../elements/Label";
+
+class Search extends Component {
+  render() {
+    //todo: add dropdown
+    return (
+      <Container className={'content__contained filter'}>
+        <Container className={'category'}>
```

Comment:
You're supposed to manage one very common technique at this point - rendering multiple uniform components ***dynamically***, using `Array.prototype.map()` method.

So, if designed appropriately, you should have separate `<CategoryFilter />` component that will accept the list of categories as one of its props and provide binding to selection action as another, i.e. something, like that (may not be precisely correct, since I didn't try that, but use just to conduct the idea):
```js
import React from 'react'

const CategoryFilter = ({categoryList, onSelection}) => {
   return !!categoryList.length && (
      <Container>
          {
             categoryList.map(category => (
                <Button 
                      onClick={onSelection}
                      key={category}
                >{category.toUpperCase()}
                </Button>
             ))
          }
      </Container>
   )
}
```

Than make use of the above within the parent component like 
```js
const categories = ['comedy', 'horror', 'drama']
const onCategorySelection = () => /* do whatever needs to be done on category chosen */
...
<CategoryFilter categoryList={categories} onSelection={onCategorySelection} />
```